### PR TITLE
Complete smart sort documentation

### PR DESCRIPTION
**Pull request by froveda**

### CODE REVIEW

File: api_docs/playlists.md [hunk 1]
```diff
@@ -291,8 +291,101 @@ playlist[priority] | The <strong>priority</strong> of the playlist related with
...
 
 Parameter | Function | Type
 --------- | -------- | ----
 playlist[sort_options] | An array of sort options. Only two levels of sorting are accepted, and videos are sorted first by first level, and second by second level | Array
+
+<p>Every <strong>sort_option</strong> has three fields:</p>
+
+<p>
+  <ul>
+    <li><strong>direction:</strong> The direction to sort the results. It accepts two values: <strong>asc</strong> (ascending) and <strong>desc</strong> (descending) order.</li>
```
Comment (imchrissmith) [inline api_docs/playlists.md]: let's define (ascending) as "(ascending - lowest to highest)" and (descending) as "(descending - highest to lowest)"

File: api_docs/playlists.md [hunk 2]
```diff
@@ -291,8 +291,101 @@ playlist[priority] | The <strong>priority</strong> of the playlist related with
...
+    <li><strong>sort_by:</strong> The field name which will be taken into account to sort the results. It accepts the following values: <strong>created_at</strong>, <strong>published_at</strong>, and <strong>title</strong>.</li>
+    <li><strong>sort_by_type:</strong> The type of the field which will be taken into account to sort the results. It accepts the following values: <strong>date</strong>, <strong>datetime</strong>, <strong>string</strong>, <strong>integer</strong>, <strong>array</strong>, and <strong>boolean</strong>.</li>
```
Comment (imchrissmith) [inline api_docs/playlists.md]: can we explain the difference between "date" and "datetime" here?

File: api_docs/playlists.md (hunk 1 above)
Comment (froveda) [inline api_docs/playlists.md]: changed!

File: api_docs/playlists.md (hunk 2 above)
Comment (froveda) [inline api_docs/playlists.md]: done!

**Pull request merged**